code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -29,124 +26,124 @@
 		..(S,H)
 		H.setMaxHealth(S.total_health)
 
-/datum/trait/nonconductive
+/datum/trait/positive/nonconductive
 	name = "Non-Conductive"
-	desc = "Decreases your susceptibility to electric shocks by a 25% amount."
-	cost = 2 //This effects tasers!
-	var_changes = list("siemens_coefficient" = 0.75)
+	desc = "Decreases your susceptibility to electric shocks by a 10% amount."
+	cost = 1 //This effects tasers!
+	var_changes = list("siemens_coefficient" = 0.9)
 
-/datum/trait/nonconductive_plus
+/datum/trait/positive/nonconductive_plus
 	name = "Major Non-Conductive"
-	desc = "Decreases your susceptibility to electric shocks by a 50% amount."
-	cost = 3 //Let us not forget this effects tasers!
-	var_changes = list("siemens_coefficient" = 0.5)
+	desc = "Decreases your susceptibility to electric shocks by a 25% amount."
+	cost = 2 //Let us not forget this effects tasers!
+	var_changes = list("siemens_coefficient" = 0.75)
 
-/datum/trait/darksight
+/datum/trait/positive/darksight
 	name = "Darksight"
 	desc = "Allows you to see a short distance in the dark."
 	cost = 1
-	var_changes = list("darksight" = 5, "flash_mod" = 2.0)
+	var_changes = list("darksight" = 4, "flash_mod" = 1.1)
 
-/datum/trait/darksight_plus
+/datum/trait/positive/darksight_plus
 	name = "Darksight (Major)"
 	desc = "Allows you to see in the dark for the whole screen."
 	cost = 2
-	var_changes = list("darksight" = 8, "flash_mod" = 3.0)
+	var_changes = list("darksight" = 7, "flash_mod" = 1.2)
```

Comment:
Also, pretty sure this is gonna make darksight not work at the edge of the screen. It's 7 tiles from the center to the edge of the screen, but would 1 mean you could see adjacent to yourself, or just the tile you're on? If it counts the center, then it should be 8.

Reply:
Ye, that was intentional. Major Darksight was not intended to be perfect darksight.

Reply:
Are  you sure about that? The description disagrees with you.